pkg/setting/setting.go

```diff
@@ -724,7 +718,7 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	cfg.IsEnterprise = IsEnterprise
 	cfg.Packaging = Packaging
 
-	cfg.ErrTemplateName = ErrTemplateName
+	cfg.ErrTemplateName = "error"
```

Comment:
Nit. Converting this from a constant is a bit weird when it cannot be changed, but no biggie.

Reply:
@marefr It's originally not a constant, but a [package global variable](https://github.com/grafana/grafana/pull/29644/files/1bbdf27d70fa22a79b005daf4ae4ed66f4eae202#diff-dc899ea32f268c53e94f76ae5ff45a239dde212d815e715fc3804ea4dfe87a75L44).